v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
+    {
+      "title": "Email password Login",
+      "icon": "emailPass",
+      "path": "/docs/emailpassword/introduction",
+      "img": "emailPassSS"
+    },
+    {
+      "title": "Social Login",
+      "icon": "social",
+      "path": "/docs/thirdparty/introduction",
+      "img": "socialSS"
+    },
+    {
```

Comment:
The icon size is not as per designs

<img width="217" alt="Screenshot 2023-03-13 at 6 52 56 PM" src="https://user-images.githubusercontent.com/18233774/224714466-ac05b72b-4163-4574-84b9-ed0de1e04730.png">